Makefile

```diff
@@ -56,6 +56,18 @@ mock:
 kotsadm:
 	go build ${LDFLAGS} ${GCFLAGS} -o bin/kotsadm $(BUILDFLAGS) ./cmd/kotsadm
 
+# Debugging
+.PHONY: kotsadm-debug-build
+kotsadm-debug-build:
+	go build ${LDFLAGS} -gcflags="all=-N -l" \
+ 		$(BUILDFLAGS) \
+		-o ./bin/kotsadm-api-debug \
+		./cmd/kotsadm
+
+.PHONY: kotsadm-debug
+kotsadm-debug: kotsadm-debug-build
+	dlv --listen=:2345 --headless=true --api-version=2 exec -- ./bin/kotsadm-api-debug
```

Comment:
https://github.com/replicatedhq/kots/blob/749cceb797f076694b52a31570430540f5c3ca3a/hack/dev/entrypoint.sh#L4
Still building, but I think the dlv command needs `api` as args in end to start the `api` server